Tasks/ServiceFabricUpdateManifests/Update-ApplicationVersions.psm1

```diff
@@ -57,20 +62,26 @@
 
                 Write-Host (Get-VstsLocString -Key SearchingApplicationType -ArgumentList $appTypeName)
 
-                $oldAppPackagePath = Join-Path $oldDropLocation $newAppPackagePath.SubString((Get-VstsTaskVariable -Name Build.SourcesDirectory -Require).Length + 1)
+                if (-not $newAppPackagePath.StartsWith($pkgArtifactPath))
+                {
+                    Write-Error (Get-VstsLocString -Key PackageIsNotUnderArtifact -ArgumentList @($newAppPackagePath, $pkgArtifactPath))
```

Comment:
No need to return?

Reply:
Write-Error effectively throws an exception an halts execution